tests/LineBot.Tests/LineBotTests/TheGetEventsMethod.cs

```diff
@@ -41,6 +41,21 @@ public async Task ShouldLogRequest()
 
                 Assert.AreEqual(expected, actual);
             }
+
+            [TestMethod]
+            public async Task ShouldHaveDestination()
+            {
+                var bot = TestConfiguration.CreateBot();
+                var request = new TestHttpRequest(JsonDocuments.Events.Webhook);
+
+                var events = await bot.GetEvents(request);
+
+                var actual = events.Destination;
+
+                var expected = "xxxxxxxxxx";
```

Comment:
Don't think we need the `actual` and `expected` variables here.

Reply:
Well, I added them because they are really helpful when debugging the tests.